firebase-installations/src/androidTest/java/com/google/firebase/installations/FirebaseInstallationsInstrumentedTest.java

```diff
@@ -117,6 +118,16 @@
           .setRegistrationStatus(PersistedFid.RegistrationStatus.UNREGISTERED)
           .build();
 
+  private static final PersistedFidEntry INVALID_FID_ENTRY =
```

Comment:
I have the feeling we talked about this multiple times already and thus please feel free to ignore:
Why is this not called PersistedInstallationEntry or something.
Since the data stored identifies the Firebase Installation as a whole, i.e. contains the Installation's FID, registration-status, it's Refresh-Token and latest Auth-Token...

Please ignore if we have discussed this already in the past.
I'm also happy to talk in person if you prefer.

Reply:
Yes, we have. I acknowledged the comments and changed to PersistedFidEntry. I can change it to 'PersistedInstallationEntry' once more as per your suggestion. But will do that as part of another PR to contain the scope of the current PR.